src/test/java/uk/gov/hmcts/sptribs/citizen/event/CicUpdateCaseEventTest.java

```diff
@@ -92,13 +93,12 @@ void shouldAddConfigurationToConfigBuilder() {
     }
 
     @Test
-    void shouldUpdateCaseDetails() {
```

Comment:
Arrays.asList(cicAppDetail) can be changed to List.of(cicAppDetail)

Reply:
Created https://tools.hmcts.net/jira/browse/DTSSTCI-366 to resolve this.